_posts/blog/2019-06-20-Hello!I am Yini He.md

```diff
@@ -14,63 +14,90 @@ author: Yini-He
 
 ## Who am I
 
---I am who I am（我就是我）
++>I am who I am（我就是我）
 
-My name is Yini He（何旖旎）.I’m a PhD student with a major in neuroimaging at the University of Electronic Science and Technology of China (UESTC).It is truly amazing for me to study in the WhitakerLab at the Alan Turing Institute as a visiting student,jointly funded by Chinese Scholarship Council (CSC) and the Newton Fund.I am enthusiastic about  the relationship between individual traits, the brain and emotions, and particularly the associated risks of developing mental disorders.
+My name is Yini He（何旖旎）.
+I’m a PhD student with a major in neuroimaging at the University of Electronic Science and Technology of China (UESTC).
+It is truly amazing for me to study in the WhitakerLab at the Alan Turing Institute as a visiting student,jointly funded by the Chinese Scholarship Council (CSC) and the Newton Fund.
+I am enthusiastic about the relationship between individual traits, the brain and emotions, and particularly the associated risks of developing mental disorders.
 
 
-## Where I come here  
+## Where I come from  
 
---As different as night and day（冰火两重天）
++>As different as night and day（冰火两重天）
 
-I was born in Changsha city of China which famous for the spicy dishes and excessively distinct seasons. And then I moved to Haikou, which famous for the gentlest sea wind and spring-like seasons. As its title indicates, these two cities have extreme difference, just like a song plays from a gentle and poetic melody to passionate and fiery bursts, fully influencing the two sides of my personality. I am happy and passionate at work, at the same time, enjoy a totally peaceful and quiet life. 
+I was born in Changsha city in China, which is famous for the spicy dishes and excessively distinct seasons.
+Then I moved to Haikou, which is famous for the gentlest sea wind and spring-like seasons. 
+As its title indicates, these two cities have extreme differences, just like a song changing from a gentle and poetic melody to passionate and fiery bursts, fully influencing the two sides of my personality. 
+I am happy and passionate at work, and at the same time I enjoy a totally peaceful and quiet life when I'm not working. 
 
 <figure>
-  <img src="/images/YiniHe/1coconut4.jpg">
+  <img src="/images/YiniHe/1coconut5.jpg">
+  <figcaption> Cold drinks and spicy food are best partners!!!(The source of picture is come from internet) </figcaption>
```

Comment:
Add link to source.

Reply:
This is composite diagram I made with Internet resources,hence there is no direct web source.

Reply:
You can link to the individual pictures you've used.